packages/react-components/react-button/src/components/Button/useButton.ts

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { ARIAButtonSlotProps, useARIAButtonShorthand } from '@fluentui/react-aria';
 import { getNativeElementProps, resolveShorthand } from '@fluentui/react-utilities';
 import type { ButtonProps, ButtonState } from './Button.types';
+import { useButtonContext } from '../../contexts/ButtonContext';
```

Comment:
_nit:_ Keep `type` imports at the bottom

```suggestion
import { useButtonContext } from '../../contexts/ButtonContext';
import type { ButtonProps, ButtonState } from './Button.types';
```